frontend/src/common/version.tsx

```diff
@@ -0,0 +1,59 @@
+import { Tooltip } from '../components/tooltip';
+import './version';
+// eslint-disable-next-line import/no-unresolved
+import version from 'virtual:git-version';
+
+function formatDate(date: string): string {
+  return new Date(date).toLocaleString('de-DE', { dateStyle: 'short', timeStyle: 'short' });
+}
+
+export function Version() {
+  const { lastCommitOnMain, branch, lastCommit, diffShort } = version;
+
+  let versionText = (
+    <>
+      <Tooltip
+        className="inline-block underline decoration-dashed"
+        tooltipText={`until ${lastCommitOnMain.hash.substring(0, 10)} from ${formatDate(
```

Comment:
Maybe a weird idea, but making this a link to github with the correct revision would be fun.
We could even implement a way to download the source that way.